core/state/access_witness.go

```diff
@@ -145,7 +145,11 @@ func (aw *AccessWitness) TouchTxOriginAndComputeGas(originAddr []byte) uint64 {
 	gas += aw.TouchAddressOnReadAndComputeGas(originAddr, zeroTreeIndex, utils.CodeKeccakLeafKey)
 	gas += aw.TouchAddressOnWriteAndComputeGas(originAddr, zeroTreeIndex, utils.NonceLeafKey)
 	gas += aw.TouchAddressOnWriteAndComputeGas(originAddr, zeroTreeIndex, utils.BalanceLeafKey)
-	return gas
+
+	// Kaustinen note: we're currently experimenting with stop chargin gas for the origin address
+	// so simple transfer still take 21000 gas. This is to potentially avoid breaking existing tooling.
+	// This is the reason why we return 0 instead of `gas`.
+	return 0
```

Comment:
Since this change isn't part of the spec yet, I preferred to leave the method implemented but just fix the returned value.

I've added this comment for any other reader wondering wth is happening here, and make it easy if we need to revert.

Reply:
that makes sense, but could you please then comment the lines above as well, so that it doesn't kill the linter?